doc/conf.py

```diff
@@ -109,7 +109,6 @@
 
 intersphinx_mapping = {
     'python': ('https://docs.python.org/3/', None),
-    'requests': ('https://docs.python-requests.org/en/latest/', None),
```

Comment:
Why removing this?

Reply:
As I made the authentication link inline, and nothing else used interphinx requests.

A

Reply:
I feel it's better to update this URL by new one (refs: #10484). What do you think?

Reply:
I don't mind either way, just important for CI to be fixed.

A

Reply:
Thanks, I'd like to choose #10484.